interface/forms/gad7/new.php

```diff
@@ -0,0 +1,219 @@
+<?php
+
+/**
+ * gad-7 form using forms api     new.php    create a new form
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    ruth moulton
+ *
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("gad7.inc"); //common strings, other includes etc
+use OpenEMR\Common\Csrf\CsrfUtils;    // security module
+formHeader("Form: GAD-7");
+$returnurl = 'encounter_top.php';   // update top of encounter window when done
```

Comment:
not seeing this used at all so safe to remove, think it's for the old frames based interface